### PR TITLE
[FLOSS H2] docs: Fix some typos and wording in docs/GETSTARTED.md

### PR DESCRIPTION
**Pull request by sedadas**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
Thank you for the very careful PR! :heart:

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by lawli3t** [comment]
Good catch, build servers should be happy when you rebase!

File: doc/GETSTARTED.md [hunk 1]
```diff
@@ -12,20 +12,20 @@
...
 
-  - We need also your skill set to improve Elektra
+  - We also need your skill set to improve Elektra
 
     You can contribute to Elektra to improve the source code, website, documentation, translation etc.
 
 - Software requirements
 
-  We need to install some basic tools to run Elektra: cmake , git , and essential build tools (make, gcc, and some standard Unix tools; alternatively [ninja](https://ninja-build.org/) and [clang](https://clang.llvm.org/index.html) are also supported but not described here). Depending on your linux distribution use following commands to install these tools:
+  We need to install some basic tools to run Elektra: cmake , git , and essential build tools (make, gcc, and some standard Unix tools; alternatively [ninja](https://ninja-build.org/) and [clang](https://clang.llvm.org/index.html) are also supported but not described here). Depending on your Linux distribution use following commands to install these tools:
```
Comment (Nirnberger) [inline doc/GETSTARTED.md:28]:
```suggestion
  We need to install some basic tools to run Elektra: cmake , git , and essential build tools (make, gcc, and some standard Unix tools; alternatively [ninja](https://ninja-build.org/) and [clang](https://clang.llvm.org/index.html) are also supported but not described here). Depending on your Linux distribution use the following commands to install these tools:
```

File: doc/GETSTARTED.md [hunk 2]
```diff
@@ -37,7 +37,7 @@
   sudo yum install -y cmake git gcc-c++
   ```
 
-  Or on macOS, most of the build tools can be obtained by installing [Xcode](https://developer.apple.com/xcode/). Other required tools may be installed using [brew ](https://brew.sh/). First install brew as described on their website. Then issue the following command to get cmake to complete the basic requirements:
+  Or on macOS, most of the build tools can be obtained by installing [Xcode](https://developer.apple.com/xcode/). Other required tools may be installed using [brew](https://brew.sh/). First install brew as described on their website. Then issue the following command to get cmake to complete the basic requirements:
```
Comment (Nirnberger) [inline doc/GETSTARTED.md:40]:
```suggestion
  Or on macOS, most of the build tools can be obtained by installing [Xcode](https://developer.apple.com/xcode/). Other required tools may be installed using [brew](https://brew.sh/). First, install brew as described on their website. Then issue the following command to get cmake to complete the basic requirements:
```

**Review by Nirnberger — APPROVED** [review]
Great work! :+1:  Very good addition/correction to the documentation. :100:

File: doc/GETSTARTED.md (hunk 1 above)
Comment (markus2330) [reply doc/GETSTARTED.md:28]:
```suggestion
  We need to install some basic tools to run Elektra: cmake, git, and essential build tools (make, gcc, and some standard Unix tools; alternatively [ninja](https://ninja-build.org/) and [clang](https://clang.llvm.org/index.html) are also supported but not described here). Depending on your Linux distribution use the following commands to install these tools:
```

**Review by markus2330 — APPROVED** [review]
Good improvements! :sparkle:

**Comment by markus2330** [comment]
Please rebase, there is a conflicting file. If possible avoid merges from master and put your commits on top with the rebase or cherry-pick.

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
Thank you for the fix! :1st_place_medal:

**Pull request merged**